app/javascript/packs/Restaurant.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types'
+
+class Restaurant extends React.Component {
+
+    render() {
+        const {restaurant} = this.props;
+        let tenbis = "";
+        if (restaurant.tenbis) {
+            tenbis = <img src="../../assets/tenbis_img.jpg" height="40" width="40"/>
```

Comment:
It's better to have height & width set in the stylesheet, not here

Reply:
Actually its better to have `height` and `width` as HTML attributes on `img` tags. That prevents hiccup on the rendering height while the image is fetching.